.github/workflows/test.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     secrets: inherit
     with:
       with_android: true
+      with_wasm: true
```

Comment:
Note: We need to wait to merge this PR until all dependencies are merged/released. I'll resolve this issue once that is done.